### PR TITLE
[database watcher] Update date/time format, add columns to session summary grid

### PR DESCRIPTION
**Pull request by dimitri-furman**
## PR Checklist

- Use the recently introduced short date/time format without milliseconds to avoid unnecessary details.
- Add the first login age and last request age columns to the session summary grid to help find long-open sessions and idle sessions.

image

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [x] Validate your changes using one or more of the [testing](../Documentation/Testing.md) methods.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [x] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Database watcher/Azure SQL Database/database/sessions/sessions.workbook [hunk 1]
```diff
@@ -552,6 +552,20 @@
                 "tooltip": "The total network IO operations for the database engine writing to the network"
               }
             },
+            {
+              "columnMatch": "first_login_age",
+              "formatter": 0,
+              "tooltipFormat": {
+                "tooltip": "Elapsed time since the earliest login time for this group of sessions, at the time of the last data collection sample. Shown as days.hours:minutes:seconds.."
```
Comment (abbyzahne) [inline Workbooks/Database watcher/Azure SQL Database/database/sessions/sessions.workbook:559]: extra `.`?

**Review by masree — APPROVED** [review]

File: Workbooks/Database watcher/Azure SQL Managed Instance/instance/sessions/sessions.workbook [hunk 2]
```diff
@@ -617,6 +617,41 @@
                 "tooltip": "The total network IO operations for the database engine writing to the network"
               }
             },
+            {
+              "columnMatch": "first_login_age",
+              "formatter": 0,
+              "tooltipFormat": {
+                "tooltip": "Elapsed time since the earliest login time for this group of sessions, at the time of the last data collection sample. Shown as days.hours:minutes:seconds.."
```
Comment (abbyzahne) [inline Workbooks/Database watcher/Azure SQL Managed Instance/instance/sessions/sessions.workbook:624]: extra `.`?

File: Workbooks/Database watcher/Azure SQL Database/database/sessions/sessions.workbook (hunk 1 above)
Comment (dimitri-furman) [reply Workbooks/Database watcher/Azure SQL Database/database/sessions/sessions.workbook]: Thanks, fixed in latest

File: Workbooks/Database watcher/Azure SQL Managed Instance/instance/sessions/sessions.workbook (hunk 2 above)
Comment (dimitri-furman) [reply Workbooks/Database watcher/Azure SQL Managed Instance/instance/sessions/sessions.workbook]: Thanks, fixed in latest

**Review by abbyzahne — APPROVED** [review]

**Pull request merged**